doc/install.text

```diff
@@ -0,0 +1,197 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+The main goal of this quick-start guide is to enable the user to extract and 
+wavelength calibrate an Arc spectrum.
+
+* Install the LSST binary distribution >= 12.0 (https://pipelines.lsst.io/install/conda.html)
+  Here it should not matter whether you install anaconda or miniconda. Please note that
+  there's only a need to install lsst-distrib, not lsst-sims.
+  Don’t forget to setup the LSST environment as mentioned on the website. You will need
+  to do this every time you want to use the pipeline in a new shell, so you might want
+  to add these 2 lines to your .bashrc:
+
+source activate lsst
+source eups-setups.sh
+
+
+* The test data are quite large (~38 MB) and are stored in git using git-lfs. We therefore
+  need to install git-lfs (https://git-lfs.github.com/). Please follow the instructions at
+  https://git-lfs.github.com to installing git-lfs (e.g. using homebrew on os/x), then type
+  
+  git lfs install
+
+  There's no need to issue any "git lfs track" commands
+
+
+* On Linux, all the anaconda packages were compiled with gcc 4.small and are incompatible
+  with modern versions. Therefore, if you are installing the PFS DRP on a Linux machine, 
+  install gcc in anaconda:
+
+conda install gcc
+
+
+* For convenience of this quick-start guide, create a directory for the PFS-DRP repositories:
```

Comment:
I think at least part of RHL's request here was not just that you explain that you wanted a working directory (which is arguably obvious) but that you're creating a particular environment variable to refer to it it — there's nothing special about the name ``$DRP_STELLA`` or ``$PFS_DRP`` except that it's convenient and you'll refer to it below. I think you could still make that clearer.